spec/services/back_end_service_spec.rb

```diff
@@ -41,4 +41,17 @@
   it 'can parse searched gyms json', :vcr do
     expect(BackEndService.gyms_near_user('80227').class).to eq(Hash)
   end
+
+  it 'can parse one gym json', :vcr do
+    result = BackEndService.get_one_gym('BJBXzKYxQAXZKb5W6HrRnA')
+    expect(result.class).to eq(Hash)
+  end
+
+  it 'can create a gym membership', :vcr do
+    json_response = File.read('./spec/fixtures/gym_membership.json')
+
+    allow(BackEndService).to receive(:post_gym_membership).and_return(JSON.parse(json_response, symbolize_names: true))
+
+    expect(BackEndService.post_gym_membership(json_response).class).to eq(Hash)
```

Comment:
Nice work on these!  I was really struggling to get passing (and tested) methods from some of the mocks and stubs